src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

Reply:
how about this comment?
https://github.com/mate-academy/jv-oop-advanced/pull/145/files#r644663221